README.md

```diff
@@ -606,6 +606,83 @@ This method will continuously execute cairo steps until the end pc, returned by
     //TODO
 ```
 
+*Step*
+
+```go
+    //TODO
+```
+
+*Decode instruction*
+
+```go
+    //TODO
+```
+
+*Run instruction*
+
+```go
+    //TODO
+```
+
+*Compute operands*
+
+```go
+    //TODO
+```
+
+*Opcode assertions*
+
+Once we have the instruction's operands to work with, we have to ensure the correctness of them. The first thing we need to differenciate is which type of instruction are we running, we do this by switching the `opcode`. 
```

Comment:
> we do this by switching the `opcode`. 
This line is confusing, we could instead say, by looking at the instruction's opcode

Reply:
Ty!